context/app/static/js/components/genes/GenePageTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Skeleton from '@mui/material/Skeleton';
+import Stack from '@mui/material/Stack';
+
+import PageTitle from 'js/shared-styles/pages/PageTitle';
+
+import { useGeneDetails, useGenePageContext } from './hooks';
+
+function GeneSymbol() {
+  const { geneSymbol } = useGenePageContext();
+  return <>{geneSymbol.toUpperCase()}</>;
```

Comment:
Now that you mention it, I have seen some gene symbols with lowercase letters as well. I've adjusted this so that it shows the approved symbol returned by the API after the data is loaded, falling back to the uppercased URL param.